Sources/Networking/HTTPClient.swift

```diff
@@ -124,7 +124,7 @@ private extension HTTPClient {
             "X-Platform-Flavor": systemInfo.platformFlavor,
             "X-Client-Version": SystemInfo.appVersion,
             "X-Client-Build-Version": SystemInfo.buildVersion,
-            "X-StoreKit2-Enabled": "\(self.systemInfo.useStoreKit2IfAvailable)",
+            "X-StoreKit2-Enabled": "\(self.systemInfo.storeKit2Setting == .enabledIfAvailable)",
```

Comment:
Should this be `self.systemInfo.storeKit2Setting == .enabledIfAvailable || self.systemInfo.storeKit2Setting == . enabledOnlyForOptimizations ` ?

Reply:
See the other comment.

Reply:
this is a debug value for us as far as I know, so we don't have to even make it a bool. 
We could just send a new header that will be one of the 3 possible values

Reply:
Oh great point!